.github/workflows/ci.yml

```diff
@@ -16,3 +16,6 @@ jobs:
       - run: npm ci
       - run: npm run lint
       - run: npm run test
+concurrency:
```

Comment:
I'd do this at the top of the file, so it doesn't get lost if you add more jobs.